tests/integ/modin/series/test_loc.py

```diff
@@ -769,13 +775,19 @@ def loc_set_helper(s):
             _item = try_convert_index_to_native(_item)
         s.loc[_row_key] = _item
 
-    with SqlCounter(query_count=1, join_count=expected_join_count):
+    query_count = 1
+    if item_type.startswith("index") and key_type.startswith("index"):
+        query_count = 7
+    elif item_type.startswith("index"):
+        query_count = 5
+    elif key_type.startswith("index"):
+        query_count = 3
+    with SqlCounter(query_count=query_count, join_count=expected_join_count):
```

Comment:
that's a wacky set of query count numbers. Do you know why it gets so high?

Reply:
added a comment explaining it, its obviously not ideal but with the current implementation it does make sense